Mathlib/Analysis/InnerProductSpace/l2Space.lean

```diff
@@ -240,7 +240,6 @@ protected theorem linearIsometry_apply_single {i : ι} (x : G i) :
   · simp [h]
 #align orthogonal_family.linear_isometry_apply_single OrthogonalFamily.linearIsometry_apply_single
 
-@[simp]
```

Comment:
Why drop this `simp`?

Reply:
`simp` proves it, and `#lint` fails on the `simpNF` linter. I don't know why CI doesn't catch it.